src/app/common/stores/applications/SsvWeb/Cluster.store.ts

```diff
@@ -144,8 +145,15 @@ class ClusterStore extends BaseStore {
       burnRate: burnRate,
       balance: newBalance,
     });
+
+    const keys = Object.keys(cluster);
+    let camelKeysCluster: any = {};
+    keys.forEach((key: string) => {
+      camelKeysCluster[_.camelCase(key)] = cluster[key];
```

Comment:
You probably need to delete the old non camel case keys

Reply:
no, I dont use in future cluster variable so don't need to do something with it